README.md

```diff
@@ -145,6 +146,7 @@ Tested with
 - Chrome 119, Chrome 119 for Android, Chrome 119 for iOS
 - Safari 17, Safari for iOS 16
 - Firefox 118, Firefox 120 for Android, Firefox 119 for iOS
+- Opera 107 for Linux, Opera 95 for Windows 7, Opera Mini for Android
```

Comment:
I'm not planning to support Opera 95 and Opera Mini.

I've documented compatibility with Opera 108 though, and I've added your documentation to the list of examples! Thanks for sharing it! 😊